lib/daru.rb

```diff
@@ -58,11 +61,21 @@ def create_has_library(library)
         class_variable_get(cv)
       end
     end
+
+    def plotting_library= lib
+      case lib
+      when :gruff, :nyaplot
+        @plotting_library = lib
+      else
+        raise ArgumentError, "Unsupported library #{lib}"
+      end
+    end
```

Comment:
I think the `plotting_library` thing actually makes sense. Since it exists for particular DFs or Vectors it's also flexible at the same time being specific. You can change your plotting library at any time and switch between interactive and static graphs.

I think this functionality will come in handy when we eventually create a library neutral plotting interface for daru, where setting `plotting_library` might  somehow allow users to leverage specific library features.